vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.cc

```diff
@@ -17,6 +17,7 @@
 #include "bat/ledger/internal/bat_contribution.h"
 #include "bat/ledger/internal/ledger_impl.h"
 #include "bat/ledger/internal/rapidjson_bat_helper.h"
+#include "brave/common/random.h"
```

Comment:
we shouldn't have `brave` dependencies inside ledger - see comment above and also any headers that are added need corresponding deps added to the gn file for the build

Reply:
so if we move this to `vendor/base/random` then you would add `//brave/vendor/base/random` to `bat-native-ledger/BUILD.gn`